docs/DeveloperGuide.md

```diff
@@ -963,10 +981,10 @@ The `organise` command will set a meet up with the time and place for all partic
 
 <div markdown="block" class="alert alert-info">
 
-:information_soruce: **Command Formats:** <br>
+:information_source: **Command Formats:** <br>
 
 * `organise INDEX` : Takes the recommendation at the specified `INDEX` and saves it.
-* `organise INDEX_1 INDEX_2 ... d/DAY T/START_TIME END_TIME l/LOCATION` : Creates a new meet up with the specified participants and information.
+* `organise INDEX_1 INDEX_2 … d/DAY T/START_TIME END_TIME l/LOCATION` : Creates a new meet up with the specified participants and information.
```

Comment:
Firstly, the `. . .`

Secondly, it would be better to be more _specific_ with the type of index. The first _index_ refers to `RecommendationIndex`, the second _index_ refers to `ContactIndex`

Reply:
Meanwhile, for `unorganise`, the `INDEX` refers to `MeetUpIndex`